Gemfile

```diff
@@ -28,3 +28,5 @@ if ENV['MOCHA_GENERATE_DOCS']
   gem 'redcarpet'
   gem 'yard'
 end
+
+gem 'ruby2_keywords', '~> 0.0.5'
```

Comment:
This is supposed to work fine with Ruby > 2.0

Reply:
👍

Reply:
> This is supposed to work fine with Ruby > 2.0

I see from the gem docs:

> Provides empty `Module#ruby2_keywords` method, for the forward source-level compatibility against ruby2.7 and ruby3.

So as I understand this,  it "works fine"  because `Module#ruby2_keywords` is a no-op in Ruby v2.0 - v2.6; but the real implementation kicks in in Ruby >= v2.7. Is that correct...?

Reply:
Yup! The gem monkey patches the method onto a few classes https://github.com/ruby/ruby2_keywords/blob/master/lib/ruby2_keywords.rb